public/index.html

```diff
@@ -1,21 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```

Comment:
when you merge your other PR you can merge master into this branch to get all of this up to date